tracks/python/exercises/raindrops/mentoring.md

```diff
@@ -0,0 +1,127 @@
+### Problem and challenges
+
+The problem asks us to take an integer and return a string.
+We need to map potential factors 3, 5, and 7 to Pling, Plang, and Plong.
+If the integer is not divisible by any of the factors, we must
+return a string representing the number in base 10.
+
+### Reasonable solutions
```

Comment:
```suggestion
### Reasonable Solutions
```

Reply:
(Either would work for me, as long as it's consistent in the track)

Reply:
It's a little bit of personal preference, but by now it is also for consistency. So far I've tried to suggest Title Casing for headings in this track for mentor notes, but a few might've slipped past unnoticed.

Reply:
Sorry - I've been off the grid